osu.Game.Rulesets.Taiko/Mods/TaikoModEasy.cs

```diff
@@ -1,12 +1,40 @@
 ﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+using System;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Beatmaps;
+using osu.Game.Graphics;
 using osu.Game.Rulesets.Mods;
 
 namespace osu.Game.Rulesets.Taiko.Mods
 {
-    public class TaikoModEasy : ModEasy
+    public class TaikoModEasy : Mod, IApplicableToDifficulty, IApplicableFailOverride
```

Comment:
Removing inheritance from `ModEasy` will break replay encoding (see `Ruleset.ConvertToLegacyMods`). I will be adding test coverage for that in another PR.